src/components/ReactionPicker.js

```diff
@@ -124,7 +124,7 @@ export const ReactionPicker = themed(
                               borderWidth: 1,
                             },
                           }}
-                          name={latestUser.id}
+                          name={latestUser.name}
```

Comment:
`latestUser.name || latestUser.id`

name is an optional field for user!! So better to have id as backup :)

Reply:
I've updated that now, thanks for the feedback